ci/environment.yml

```diff
@@ -1,7 +1,12 @@
 channels:
   - conda-forge
 dependencies:
+  - conda
   - python >=3.9
+  - pyspark ==3.4.1
+  - openjdk ==20.0.2
+  - python-duckdb ==0.9.0
```

Comment:
I think that you want 0.9.1 here.  It was released literally minutes ago, and it stops a conflict with pandas 2.1.0